docusaurus/docs/ui-extensions/create-an-extension-point.md

```diff
@@ -15,40 +15,285 @@ keywords:
   - apps
 ---
 
-Providing an extension point within your UI enables other plugins to contribute supplementary capabilities, and leverage the context within the current view to guide users to a potential next step. Here's what you need to know about extension points.
+## What is an extension point?
```

Comment:
I agree with Joe here. It's a bit funny to have stacked headers. Maybe we can do something like:

```suggestion

In this guide you will learn how to provide an extension point so that other app plugins can add their extensions to your plugin.

## What is an extension point?
```